Marlin/src/feature/touch/xpt2046.cpp

```diff
@@ -23,7 +23,10 @@
 
 #include "xpt2046.h"
 #include "../../inc/MarlinConfig.h"
-#include "../../lcd/dogm/ultralcd_DOGM.h" // for LCD_FULL_PIXEL_WIDTH, etc.
+#if ENABLED(HAS_GRAPHICAL_LCD)
+  #include "../../lcd/dogm/ultralcd_DOGM.h" // for LCD_FULL_PIXEL_WIDTH, etc.
+#endif
```

Comment:
Please test with `FSMC_GRAPHICAL_TFT` which is the reason for this include having no wrapper.

Reply:
For `TFT_LVGL_UI` will work, because I just want remove that include, so it dont depends on U8glib...
I used `HAS_GRAPHICAL_LCD`, because I dont know if other that `FSMC_GRAPHICAL_TFT` define needs to use xpt2046...
But, thinking more about it, TOUCH_BUTTONS only working in a FSMC_GRAPHICAL_TFT display, right?
So we can change it.

Reply:
The `TOUCH_BUTTONS` feature is specific to the XPT2046, which may only be installed on certain displays, and for future compatibility with whatever those displays may be it includes the DOGM header to get the screen size defines.